packages/gatsby/src/utils/webpack.config.js

```diff
@@ -451,47 +451,61 @@ module.exports = async (program, directory, suppliedStage) => {
   }
 
   if (stage === `build-html` || stage === `develop-html`) {
+    // Packages we want to externalize to save space
     const externalList = [
-      // match `lodash` and `lodash/foo`
-      // but not things like `lodash-es`
+      `common-tags`,
+      /^core-js\//,
+      `crypto`,
+      `debug`,
+      `fs`,
+      `https`,
+      `http`,
       `lodash`,
+      `path`,
+      `semver`,
       /^lodash\//,
+      `zlib`,
+    ]
+
+    // Packages we want to externalize because the user should provide them
+    const userExternalList = [
+      `es6-promise`,
+      `minimatch`,
+      `pify`,
+      `react-helmet`,
       `react`,
       /^react-dom\//,
-      `pify`,
-      `@reach/router`,
+      `@react/router`,
```

Comment:
this should be @reach/router and it also should be in main externalList (not "optional" ) as this is hard dependency in gatsby (at least right now)

Reply:
Oops typo! Fixed 👍